docroot/modules/custom/foia_file/foia_file.drush.inc

```diff
@@ -0,0 +1,108 @@
+<?php
+
+/**
+ * @file
+ * FOIA Api module drush command.
+ */
+
+use Drupal\file_entity\Entity\FileEntity;
+
+/**
+ * Implements hook_drush_command().
+ */
+function foia_file_drush_command() {
+  $items = [];
+
+  $items['file-entity-update'] = [
+    'description' => 'Updates file entities virus scan statuses.',
+    'core' => ['8+'],
+    'arguments' => [
+      'scan_result' => 'The result of the file scan.',
+    ],
+    'aliases' => ['feu'],
+  ];
+
+  return $items;
+}
+
+/**
+ * Implements drush_hook_COMMAND().
+ */
+function drush_foia_file_file_entity_update($scanResult) {
+
+  $startTime = microtime(TRUE);
+
+  $message = "File Entity update started. *******************************\n";
+
+  // Get webform directory on the server.
+  $fileDir = explode('/webform/', $scanResult, 2);
+  $webformDir = "{$fileDir[0]}/webform/";
+
+  // Temporarily delete instances of the webform directory during processing.
+  $trimmedFiles = str_replace($webformDir, '', $scanResult);
+
+  // Split string into lines. @todo change to preg split \R?
+  $scans = explode("\n", $trimmedFiles);
+
+  foreach ($scans as $scan) {
+
+    // Split the scan line into filename and status.
+    preg_match('/[^ ]*$/', $scan, $wordsArray);
+    // Set status to the last word of the line.
+    $status = $wordsArray[0];
+
+    // Get relative file path.
+    $scanLength = strlen($scan);
+    $statusLength = strlen($status);
+    $fileLength = $scanLength - $statusLength - 2;
+    $fileNameWithVirusName = substr($scan, 0, $fileLength);
+
+    // Trim virus description.
+    preg_match('/(\.(pdf|txt|doc|docx|fodt|odt|gif|jpg|png))(: .*)?$/', $fileNameWithVirusName, $fileExtension);
```

Comment:
Might want to test this with files that have weird names. I'm also thinking we'll need to store these file extensions in config in case they ever change. Or, we might be able to grab the list of allowed extensions from existing config we have committed from our webform configuration.